types/int.go

```diff
@@ -214,6 +214,11 @@ func (i Int) Neg() (res Int) {
 	return Int{neg(i.i)}
 }
 
+// String converts int to string
+func (i Int) String() string {
+	return i.i.String()
```

Comment:
Does the tendermint-lint require a comment on this function? I don't think it does - pretty straight forward probably can remove the comment (unless this needs to be added to our linter)